### PR TITLE
[TESTS] Wallet unit tests

### PR DESCRIPTION
**Pull request by Bagaric**

### CODE REVIEW

File: test/unit/UnitTestWallet.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  });
+
+  describe('loadPrivateKeys()', () => {
+    it('creates a wallet from a private key', () => {
+      assert.equal(wallet.length, 0);
+
+      const privKey = 'fdf2e15fd858d9d81e31baa1fe76de9c7d49af0018a1322aa2b9e493b02afa26';
+      wallet.loadPrivateKeys([privKey]);
+
+      assert.equal(wallet.length, 1);
```
Comment (Kuzirashi) [inline test/unit/UnitTestWallet.ts]: I know that library is handling derivation of public address from private key, but could be worth to check if public address of loaded wallet is as expected.

**Comment by Bagaric** [comment]
@lsaether yeah will add those, didn't notice I skipped those.

File: test/unit/UnitTestWallet.ts [hunk 2]
```diff
@@ -122,9 +138,73 @@ describe('Wallet Unit Tests', () => {
...
+      assert.equal(txHash.length, 66);
+
+      const receipt = await wallet.sendFromIndex(idx, opts);
+      expect(receipt).to.haveOwnProperty('from');
+      expect(receipt).to.haveOwnProperty('receipt');
+    });
+  });
+
+  describe('sendFromNext()', () => {
+    it('returns generates a random index and sends from it', async () => {
```
Comment (lsaether) [inline test/unit/UnitTestWallet.ts]: The index isn't random it goes sequentially

Comment (Bagaric) [reply test/unit/UnitTestWallet.ts]: True, misunderstood that. Corrected.

**Pull request merged**